sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -150,6 +163,39 @@ export class Sender {
     sendBatch(messages: SendableMessageInfo[]): Promise<void>;
     }
 
+// @public
+export class ServiceBusAtomManagementClient extends ServiceClient {
```

Comment:
Customers will not be aware of the existence of a separate client in order to carry out the crud operations. They are aware of the existing `ServiceBusClient` and would expect all the crud operations to exist on it.

Below is what I would recommend:
- Do not expose `ServiceBusAtomManagementClient` to customer, instead have it as property on `ServiceBusClient`.
When user creates a `ServiceBusClient`, you instantiate `ServiceBusAtomManagementClient` internally 
- Replicate all the crud operations on `ServiceBusClient`. These will call the corresponding counterparts in the `ServiceBusAtomManagementClient`